CHANGELOG.asciidoc

```diff
@@ -216,6 +216,7 @@ https://github.com/elastic/beats/compare/v6.2.3...master[Check the HEAD diff]
 - Add Audit log fileset to the Elasticsearch module. {pull}7365[7365]
 - Add Slow log fileset to the Elasticsearch module. {pull}7473[7473]
 - Add deprecation fileset to the Elasticsearch module. {pull}7474[7474]
+- Add `config_timezone` option to Kafka module to convert dates to UTC. {issue}7546[7546] {pull}7578[7578]
```

Comment:
@jsoriano I think this line should say `convert_timezone`, not `config_timezone`, correct?

Reply:
Correct, I'll open a PR to fix this.

Reply:
https://github.com/elastic/beats/pull/8001